### ISSUE TITLE
Fatal Exception: java.lang.RuntimeException CONNECTIVITY_CHANGE crash

### ISSUE THREAD

**Issue by melirenia** [sematext/sematext-logsene-android#29]
There is a crash when network state changes. Mostly it happens on Android 5 devices (minSdkVersion 21 in my project).

Logs below

```
Fatal Exception: java.lang.RuntimeException: Error receiving broadcast Intent { act=android.net.conn.CONNECTIVITY_CHANGE flg=0x4000010 (has extras) } in androidx.work.impl.constraints.trackers.NetworkStateTracker$NetworkStateBroadcastReceiver@d734889
       at android.app.LoadedApk$ReceiverDispatcher$Args.run(LoadedApk.java:982)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:145)
       at android.app.ActivityThread.main(ActivityThread.java:6872)
       at java.lang.reflect.Method.invoke(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:372)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1404)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1199)
As i understand you just need to upgrade your WorkManager lib version to the latest one.

Caused by java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask@89daab0 rejected from java.util.concurrent.ScheduledThreadPoolExecutor@29437d29[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 0]
       at java.util.concurrent.ThreadPoolExecutor$AbortPolicy.rejectedExecution(ThreadPoolExecutor.java:2011)
       at java.util.concurrent.ThreadPoolExecutor.reject(ThreadPoolExecutor.java:793)
       at java.util.concurrent.ScheduledThreadPoolExecutor.delayedExecute(ScheduledThreadPoolExecutor.java:298)
       at java.util.concurrent.ScheduledThreadPoolExecutor.schedule(ScheduledThreadPoolExecutor.java:503)
       at java.util.concurrent.Executors$DelegatedScheduledExecutorService.schedule(Executors.java:644)
       at androidx.work.impl.background.systemalarm.WorkTimer.startTimer(WorkTimer.java:82)
       at androidx.work.impl.background.systemalarm.DelayMetCommandHandler.onAllConstraintsMet(DelayMetCommandHandler.java:100)
       at androidx.work.impl.constraints.WorkConstraintsTracker.onConstraintMet(WorkConstraintsTracker.java:150)
       at androidx.work.impl.constraints.controllers.ConstraintController.updateCallback(ConstraintController.java:134)
       at androidx.work.impl.constraints.controllers.ConstraintController.onConstraintChanged(ConstraintController.java:141)
       at androidx.work.impl.constraints.trackers.ConstraintTracker.setState(ConstraintTracker.java:103)
       at androidx.work.impl.constraints.trackers.NetworkStateTracker$NetworkStateBroadcastReceiver.onReceive(NetworkStateTracker.java:170)
       at android.app.LoadedApk$ReceiverDispatcher$Args.run(LoadedApk.java:972)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:145)
       at android.app.ActivityThread.main(ActivityThread.java:6872)
       at java.lang.reflect.Method.invoke(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:372)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1404)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1199)
```

This ussie as i understand is related to an old WorkManager library version which need to be updated. Here is what i found accourding this issue https://stackoverflow.com/questions/55530714/java-util-concurrent-rejectedexecutionexception-when-using-workmanager 
Crash is not happening when i disable Logsene. 
Please take a look at it.

https://github.com/sematext/sematext-logsene-android/blob/6d583cf5c5a1545919d55833a4c3e53660df9251/logseneandroid/build.gradle#L32

**Comment by otisg** [comment]
Do you have the latest version of the Logsene Android library, @Meliphant ?

**Comment by gr0** [comment]
I'll have a look into that as soon as possible @Meliphant, which probably means tomorrow. Thanks for reporting that. Of course, if you think about providing a PR it is more than welcome :)

**Comment by gr0** [comment]
Hi @Meliphant I just released the 2.5.0 version of the library. Please check it when you can.
